wp-content/plugins/quizflow/quizflow.php

```diff
@@ -0,0 +1,55 @@
+<?php 
+/**
+* Plugin Name: QuizFlow
+* Plugin URI: http://pmdesigns.ca/
+* Description: A WordPress plugin designed specifically for Batty's Bath that enables you to create quizzes like a flow chart, guiding the customer to the product that best suits their needs.
+* Version: 1.0
+* Author: Pat Monette & Ryan McHale
+* Author URI: http://pmdesigns.ca/
+*/
+
+// Register the QuizFlow Post Type
+ 
+function register_cpt_qf_quiz() {
+ 
+    $labels = array(
+        'name' => _x( 'QF Quizzes', 'qf_quiz' ),
```

Comment:
- [ ] It's great that you're thinking ahead to translations, but there's an issue here.

I'm assuming you found a tutorial for this so you might not have done this intentionally, so I'll explain. That _x function is a call to a gettext library used for translating strings. It's really good practise to wrap any text that you write from a code perspective in these kinds of functions. With _x however, the second argument it expects is the context. It's meant to help translators avoid mistranslating a word. For instance, the word 'post' can be both a noun and a verb, etc. It looks like the second parameter you're passing is actually the domain, basically just helping translators see what plugin this particular string belongs to. So _x isn't the right method to use here. We should use __() instead (double underscores and then the brackets.

You can see all these functions in ``\quizflow\wp-includes\l10n.php``, and there's a really great article on the [Wordpress Codex](https://codex.wordpress.org/I18n_for_WordPress_Developers) that explains things more in depth.

Aside from that everything looks great and works as expected, so once these are changed and updated It'll get merged.